tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1292,3 +1292,53 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     )
   });
 }
+
+describe('Test move cluster to newly created workspace and deploy application to it.', { tags: '@p1'}, () => {
+  qase(51,
+    it("Fleet-51: Test move cluster to newly created workspace and deploy application to it.", { tags: '@fleet-51' }, () => {
+      const repoName = 'default-cluster-new-workspace-51'
+      const branch = "master"
+      const path = "simple"
+      const repoUrl = "https://github.com/rancher/fleet-examples"
+      const flagName = "provisioningv2-fleet-workspace-back-population"
+      const newWorkspaceName = "new-fleet-workspace"
+      const fleetDefault = "fleet-default"
+      let timeout = 30000
+
+      if (/\/2\.11/.test(Cypress.env('rancher_version'))) {
+        timeout = 60000
```

Comment:
is this by design or to workaround the test?
if by design, ok, else, IMO we should not workaround it and open issue.

Reply:
I confirmed yesterday, it is by design after 2.11 onwards...

Reply:
Outstanding. Thanks.